src/client/pfs/pfs.proto

```diff
@@ -312,12 +312,20 @@ message TagObjectRequest {
   repeated Tag tags = 2;
 }
 
+message ListObjectsRequest {}
+
+message ListObjectsTaggedWithPrefixRequest {
+  string prefix = 1;
+}
```

Comment:
The name is definitely kinda verbose, but calling it `ListTag` doesn't seem to make sense since it returns objects.  This API basically returns objects that are tagged with tags that have a given prefix, hence the name.